WordPress/Classes/Services/ReaderTopicService.h

```diff
@@ -26,7 +28,7 @@ extern NSString * const ReaderTopicFreshlyPressedPathCommponent;
  @param success block called on a successful fetch.
  @param failure block called if there is any error. `error` can be any underlying network error.
  */
-- (void)fetchReaderMenuWithSuccess:(void (^)(void))success failure:(void (^)(NSError *error))failure;
+- (void)fetchReaderMenuWithSuccess:(void (^)(void))success failure:(void (^)(NSError * _Nullable error))failure;
```

Comment:
Did you consider making these `nonnull`? 

Or, given there are few changes to the rest of the app, it was simpler to have them nullable? Nothing that we can't change later anyway.